packages/api/login.go

```diff
@@ -122,27 +134,43 @@ func login(w http.ResponseWriter, r *http.Request, data *apiData, logger *log.En
 		founder = converter.StrToInt64(sp.Value)
 	}
 
-	result := loginResult{EcosystemID: converter.Int64ToStr(state), KeyID: converter.Int64ToStr(wallet),
-		Address: address, IsOwner: founder == wallet, IsNode: conf.Config.KeyID == wallet,
-		IsVDE: model.IsTable(fmt.Sprintf(`%d_vde_tables`, state))}
+	result := loginResult{
+		EcosystemID: converter.Int64ToStr(state),
```

Comment:
if "state" is local variable - replace it with "ecosystem"

Reply:
ecosystemID